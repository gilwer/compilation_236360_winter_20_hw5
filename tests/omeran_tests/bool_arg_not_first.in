//func args
int foo(int w, byte x, bool c, int y) {
	printi(w);
	printi(x);
	if (c) print("true"); else print("false");
	printi(y);
	return 0;
}

void main() {

	byte x = 88b;
	byte y = 0b;
	bool z = true;
	foo(x,y,z,x);
//	foo(x + 23,y - 55b,87 >= x, 123 * x * (y+2));
}