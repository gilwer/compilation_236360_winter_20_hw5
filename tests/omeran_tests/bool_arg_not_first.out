88
0
true
88
